resources/function_help/json/array_majority

```diff
@@ -2,10 +2,18 @@
   "name": "array_majority",
   "type": "function",
   "groups": ["Arrays"],
-  "description": "Returns an array containing the most common value in an array. The returned array may contain more than one value if multiple values occur equally often.",
-  "arguments": [ {"arg":"array","description":"an array"} ],
-  "examples": [ 
-    { "expression":"array_majority(array(0,1,42,42,43))", "returns":"[ 42 ]"},
-    { "expression":"array_majority(array(0,0,1,2,2,42))", "returns":"[ 0, 2 ]"}
+  "description": "Return the most common values in an array.",
+  "arguments": [
+    {"arg":"array","description":"an array"},
+    {"arg":"option='all'", "optional":true, "description":"a string specifying the return values handling. Valid options are:<br /><ul><li>all: Default, all most common values are returned in an array</li><li>first: Return the first of the most common values</li><li>median: Return the median of the most common values</li><li>real_majority: Return the value which occur more than half the size of the array</li></ul>"}
```

Comment:
I have no rewording suggestion but wonder if the `first` parameter description should not be tweaked to clarify that it's `array_majority(array(0,1,42,42,43,43,1), 'first') --> 42` (the most common value whose count first hit the "max" -- current implementation) and not
`array_majority(array(0,1,42,42,43,43,1), 'first') --> 1` (the most common value that appears first in the array)?

Reply:
```suggestion
    {"arg":"option='all'", "optional":true, "description":"a string specifying the return values handling. Valid options are:<br /><ul><li>all: Default, all most common values are returned in an array</li><li>first: Returns the first of the most common values</li><li>median: Returns the median of the most common values</li><li>real_majority: Returns the value which occurs more than half the size of the array</li></ul>"}
```